packages/fluentui-rc/src/SelectWidget/SelectWidget.tsx

```diff
@@ -81,6 +83,7 @@ function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFSchema, F extend
         selectedOptions={selectedIndexesAsArray}
         aria-describedby={ariaDescribedByIds<T>(id)}
       >
+        {!multiple && schema.default === undefined && <Option value=''>{placeholder || ''}</Option>}
```

Comment:
What happens if you remove the `| ''` and `placeholder` is undefined? I think it might be ok that way, or does `undefined` get output?

Reply:
I actually get a TypeScript error. I think the library requires that either the `text` or `children` prop is non-null

```
src/SelectWidget/SelectWidget.tsx:86:56 - error TS2322: Type '{ children: string | undefined; value: string; }' is not assignable to type 'IntrinsicAttributes & (OptionProps & RefAttributes<HTMLDivElement>)'.
  Property 'text' is missing in type '{ children: string | undefined; value: string; }' but required in type '{ text: string; children?: ReactNode; }'.

86         {!multiple && schema.default === undefined && <Option value=''>{placeholder}</Option>}
                                                          ~~~~~~

  ../../node_modules/@fluentui/react-combobox/dist/index.d.ts:301:5
    301     text: string;
            ~~~~
    'text' is declared here.
```

Reply:
I can remove similar code from the MUI SelectWidget